CONTRIBUTING.md

```diff
@@ -215,7 +215,7 @@ Must be one of the following:
 * **docs**: Documentation only changes
 * **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing
   semi-colons, etc)
-* **refactor**: A code change that neither fixes a bug or adds a feature
+* **refactor**: A code change that neither fixes a bug nor adds a feature
```

Comment:
It's just my opinion, keeping the language simpler makes it more accessible to non-english speakers. But, in this particular case it might not matter that much, so LGTM anyways --- we can fix it later if it confuses people.